pb_bss/evaluation/wrapper.py

```diff
@@ -142,24 +191,47 @@ def si_sdr(self):
                 '`enable_si_sdr=True`.'
             )
 
-    def as_dict(self):
-        metrics = dict(
-            pesq=self.pesq,
-            stoi=self.stoi,
-            mir_eval_sxr_sdr=self.mir_eval['sdr'],
-            mir_eval_sxr_sir=self.mir_eval['sir'],
-            mir_eval_sxr_sar=self.mir_eval['sar'],
-        )
-
+    def _available_metric_names(self):
+        metric_names = [
+            'pesq',
+            'stoi',
+            'mir_eval_sdr',
+            'mir_eval_sir',
+            'mir_eval_sar',
+        ]
         if self.enable_si_sdr:
-            metrics['si_sdr'] = self.si_sdr
-
+            metric_names.append('si_sdr')
         if self._has_image_signals:
-            metrics['invasive_sxr_sdr'] = self.invasive_sxr['sdr']
-            metrics['invasive_sxr_sir'] = self.invasive_sxr['sir']
-            metrics['invasive_sxr_snr'] = self.invasive_sxr['snr']
+            metric_names.append('invasive_sdr')
+            metric_names.append('invasive_snr')
+            metric_names.append('invasive_sir')
+
+        return tuple(metric_names)
+
+    def _disableded_metric_names(self):
```

Comment:
I think this is a typo.

Reply:
Thanks. I pushed a fix to the master branch.

Reply:
Hmm sorry, I should have mentioned that the usage was consistent though.

Reply:
Sorry, my fault. I was with my thoughts somewhere else.
Thanks again.